### PR TITLE
Release notes

### PR DESCRIPTION
**Pull request by markus2330**
## Basics

These points need to be fulfilled for every PR:

- [ ] Short descriptions of your changes are in the release notes
      (added as entry in `doc/news/_preparation_next_release.md` which
      contains `_(my name)_`)
      **Please always add something to the release notes.**
- [ ] Details of what you changed are in commit messages
      (first line should have `module: short statement` syntax)
- [ ] References to issues, e.g. `close #X`, are in the commit messages.
- [ ] The buildservers are happy.
- [ ] The PR is rebased with current master.

If you have any troubles fulfilling these criteria, please write
about the trouble as comment in the PR. We will help you.
But we cannot accept PRs that do not fulfill the basics.

## Checklist

Check relevant points but **please do not remove entries**.
For docu fixes, spell checking, and similar none of these points below
need to be checked.

- [ ] I added unit tests for my code
- [ ] I fully described what my PR does in the documentation
      (not in the PR description)
- [ ] I fixed all affected documentation
- [ ] I added code comments, logging, and assertions as appropriate (see [Coding Guidelines](https://master.libelektra.org/doc/CODING.md))
- [ ] I updated all meta data (e.g. README.md of plugins and [METADATA.ini](https://master.libelektra.org/doc/METADATA.ini))
- [ ] I mentioned every code not directly written by me in [THIRD-PARTY-LICENSES](doc/THIRD-PARTY-LICENSES)

## Review

Reviewers will usually check the following:

- [ ] Documentation is introductory, concise, good to read and describes everything what the PR does
- [ ] Examples are well chosen and understandable
- [ ] Code is conforming to [our Coding Guidelines](https://master.libelektra.org/doc/CODING.md)
- [ ] APIs are conforming to [our Design Guidelines](https://master.libelektra.org/doc/DESIGN.md)
- [ ] Code is consistent to [our Design Decisions](https://master.libelektra.org/doc/decisions)

## Labels

If you are already Elektra developer:

- Add the "work in progress" label if you do not want the PR to be reviewed yet.
- Add the "ready to merge" label **if the basics are fulfilled** and you also
  say that everything is ready to be merged.

### CODE REVIEW

File: doc/news/2019-11-26_0.9.1.md [hunk 1]
```diff
@@ -437,16 +417,24 @@ you up to date with the multi-language support provided by Elektra.
...
 - Renamed the sub menu _Getting Started_ in the menu _Documentation_ to _About Elektra_ and removed the sub-sub-menus _Compiling_, _Installation_ and _Tutorials_. These sub-sub-menus are now sub-menus of _Documentation_. A new sub menu labeled _Get Started_ added to the menu _Documentation_ with some newcomer-friendly informations. Renamed the _Getting Started_ sub menu in _Development_ to _Contribute to Elektra_. The green button on the main page is routed to the new _Get Started_ page. _(Hani Torabi)_
+- The website now lives in the folders [website-frontend](/src/tools/website-frontend) and [website-backend](/src/tools/website-backend) to avoid confusion with the REST backend of the Web-UI. _(Markus Raab)_
 - Improve main page of website, restructure getting started. _(Markus Raab)_
 
 ## Outlook
 
 We are currently working on following topics:
 
-- Merge tool in C99 (the same language as the core of Elektra). This is planned to supersede the existing merge tool. The goal is to reduce the number of merge conflicts in contrast to regular merge tools using the specific semantics of configuration files. _(Dominic Jäger)_
+- Elektrify KDE _(Dardan Haxhimustafa and Felix Resch)_
```
Comment (mpranj) [inline doc/news/2019-11-26_0.9.1.md]:
```suggestion
- Elektrify KDE _(Dardan Haxhimustafa)_ and _(Felix Resch)_
```

File: doc/news/2019-11-26_0.9.1.md [hunk 2]
```diff
@@ -17,9 +17,10 @@ You can also read the news [on our website](https://www.libelektra.org/news/0.9.
 
 ## Highlights
 
-- Code generation is ready for productive use. _(Klemens Böswirth)_
-- Elektra now has a technical preview of a new [merge library](../tutorials/cmerge.md) offering a number of [merge strategies](../help/elektra-cmerge-strategy.md). It is written in C99 and can currently be used with [`kdb cmerge`](../help/kdb-cmerge.md). _(Dominic Jäger)_
-- We are working on integrating Elektra into [KDE](https://kde.org/). A new Elektra plugin called `kconfig` was added which can read KDE's kconfig ini files. Additionally we are working on [a fork](https://github.com/ElektraInitiative/kconfig) of KDE's `KConfig` configuration system and patching it to use libelektra. _(Dardan Haxhimustafa and Felix Resch)_
+- We are working on integrating Elektra into [KDE](https://kde.org/). A new Elektra plugin called `kconfig` was added which can read KDE's kconfig ini files. Additionally we are working on a [fork](https://github.com/ElektraInitiative/kconfig) of KDE's `KConfig` configuration system and patching it to use libelektra. _(Dardan Haxhimustafa and Felix Resch)_
```
Comment (mpranj) [inline doc/news/2019-11-26_0.9.1.md]:
```suggestion
- We are working on integrating Elektra into [KDE](https://kde.org/). A new Elektra plugin called `kconfig` was added which can read KDE's kconfig ini files. Additionally we are working on a [fork](https://github.com/ElektraInitiative/kconfig) of KDE's `KConfig` configuration system and patching it to use libelektra. _(Dardan Haxhimustafa)_ and _(Felix Resch)_
```

File: doc/news/2019-11-26_0.9.1.md [hunk 3]
```diff
@@ -17,9 +17,10 @@ You can also read the news [on our website](https://www.libelektra.org/news/0.9.
...
+- The [elektra](https://crates.io/crates/elektra) and [elektra-sys](https://crates.io/crates/elektra-sys) crates have been published to crates.io for easier usage of our Rust binding. _(Philipp Gackstatter)_
+- We improved our [Go bindings](https://github.com/ElektraInitiative/go-elektra).
```
Comment (mpranj) [inline doc/news/2019-11-26_0.9.1.md]:
```suggestion
- We improved our [Go bindings](https://github.com/ElektraInitiative/go-elektra). _(Raphael Gruber)_
```

File: doc/news/2019-11-26_0.9.1.md [hunk 4]
```diff
@@ -378,21 +371,11 @@ you up to date with the multi-language support provided by Elektra.
...
 - Improved `range` plugin error message. _(Michael Zronek)_
 - Improved error codes documentation to clarify the hierarchy for developers. _(Michael Zronek)_
-- Out of memory error messages are now uniform. _(Michael Zronek)_
 - Release notes now use git's union merge driver. _(Dominic Jäger)_
 - Updated asciinema recording to correctly use `sudo`. _(Michael Zronek)_
-- Please remove me. I'm only here for the build server. _(Dominic Jäger)_
-- Fixed outdated docu in plugins.md. _(Michael Zronek)_
-- I'm only here for the build server. This PR contains only fix ups. _(Dominic Jäger)_
-- Please remove me. I'm only here for the build server. _(Dominic Jäger)_
-- Please remove me. I'm only here for the build server. _(Dominic Jäger)_
```
Comment (mpranj) [inline doc/news/2019-11-26_0.9.1.md]: Thank you, seems like I was blind yesterday! :medal_military:

**Comment by markus2330** [comment]
> Is it ready from your side or do you want to add anything?

There is always more what could be improved but I think it is fine now. If you publish this on the website, I can send out the notifications.

**Comment by mpranj** [comment]
Thank you so much!

**Pull request merged**